_posts/2022-04-19-network-upgrade-july-2022.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Monero will undergo a network upgrade on 16th July, 2022"
+summary: Announcement of the upcoming network upgrade (16th July, 2022)
+tags: [community, protocol upgrade, core, crypto]
+author: sethforprivacy
+image: /blog/assets/2022-04-19-network-upgrade-july-2022/upgrades.png
+---v
+
+<div align="center">
+    <img src="{{ page.image }}" width="350px">
+</div>
+
+The next network upgrade will take place on **16th July, 2022** (block 2,668,888). Network upgrades are technically hard-forks and require all participants in the Monero network to update their software. The new software will be released one month before the network upgrade, on the **~16th of June**.
+
+This network upgrade will introduce important new features like:
+
+- A ring-size increase from 11 to 16, increasing the base anonymity set (read: plausible deniability, or base privacy) of every transaction on the network
```

Comment:
A ring-size increase from 11 to 16, **the largest ever absolute increase in the base anonymity set** (read:

Reply:
Love this language, good addition.